### ISSUE TITLE
[Feature Request] Headscale 0.23 support

### ISSUE THREAD

**Issue by ventsislav-georgiev** [gabe565/charts#692]
### Chart Name

Headscale

### Is your feature request related to a problem? Please describe.

The current chart supports 0.22, but the recently released 0.23 fails.

### Describe the solution you'd like

Support for https://github.com/juanfont/headscale/releases/tag/v0.23.0

**Comment by gabe565** [comment]
You're right, a bunch of envs changed. I'll update the values and merge https://github.com/gabe565/charts/pull/679 ASAP!

**Comment by gabe565** [comment]
@ventsislav-georgiev Ok the chart has been updated. Note that you may need to change some envs for this update, notably:
- `env.HEADSCALE_DNS_CONFIG_BASE_DOMAIN` changed to `env.HEADSCALE_DNS_BASE_DOMAIN`
- `env.HEADSCALE_IP_PREFIXES` has been split into `env.HEADSCALE_PREFIXES_V4` and `env.HEADSCALE_PREFIXES_V6`
- `env.HEADSCALE_DNS_CONFIG_NAMESERVERS` changed to `env.HEADSCALE_DNS_NAMESERVERS_GLOBAL`


See [Headscale's v0.23.0 Release Notes](https://github.com/juanfont/headscale/releases/tag/v0.23.0) for a list of all changes.
